### ISSUE TITLE
How to implement RuleRequired for custom row having multiple textfields

### ISSUE THREAD

**Issue by reenaphilip** [xmartlabs/Eureka#2096]
I have created a custom row with cell having 3 textfields. Before submitting the form to server I should ensure all the three fields are filled by the user. 
How can I add Rule for this?

**Comment by mats-claassen** [comment]
RuleRequired checks whether the value is `nil` or an empty string (in case its type is String) For your custom value object I guess you will have to create a new Rule which checks that all three parts of your value are not empty. I am supposing that your value is some sort of Equatable object with three variables for those text fields?

**Comment by reenaphilip** [comment]
Yes its an object. Will check that with nil for all the three values.
